steps/create_databases.sh

```diff
@@ -7,7 +7,9 @@ set -euo pipefail
 
 (
  # Import the logging functions
-    source /opt/emr/logging.sh
+    source /opt/emr/resume_step.sh
```

Comment:
```suggestion
    source /opt/emr/retry.sh
```

was this one supposed to be the retry?